### PR TITLE
fix(#1088): combobox didn't allow aria-labels to be changed

### LINKED ISSUE #1088: Clear selection button text hard-coded in combo-box
The clear selection button text is hard-coded in combo-box,
No way to give custom text.

image

### PR DESCRIPTION
**Pull request by rapzo**
Fixes #1088 

Change-log:
- all aria-labels and some titles were hard-coded
- dropdown items didn't highlight title (as its content)
- combobox story with custom templates was not testable

### CODE REVIEW

File: src/combobox/combobox.component.ts [hunk 1]
```diff
@@ -186,6 +185,30 @@ export class ComboBox implements OnChanges, AfterViewInit, AfterContentInit {
...
+	 */
+	@Input() clearSelectionsAria = "Clear all selected items";
+	/**
+	 * Value to display on the clear the selected item icon, when single is selected
+	 */
+	@Input() clearSelectionTitle = "Clear selected item";
+	/**
+	 * Value to display for accessibility purposes on the clear the selected item icon, when single is selected
+	 */
+	@Input() clearSelectionAria = "Clear Selection";
```
Comment (Donisius) [inline src/combobox/combobox.component.ts]: We should add these into translatable text, similar to the way the number component does this, check out https://github.com/IBM/carbon-components-angular/blob/9b1d712fcb9d269155dbe14e2c25c544183a865f/src/number-input/number.component.ts#L177-L198

Comment (rapzo) [reply src/combobox/combobox.component.ts]: Let me know if it's how you wanted.

**Comment by rapzo** [comment]
Glad to help!
Was looking at the board and decided to lend a hand. Let me know if you want a specific order, otherwise i might grab some other issue.

File: src/combobox/combobox.component.ts (hunk 1 above)
Comment (Donisius) [reply src/combobox/combobox.component.ts]: Perfect!

File: src/i18n/en.ts [hunk 2]
```diff
@@ -55,6 +55,17 @@ export default {
 		"COPY_CODE": "Copy code",
 		"COPIED": "Copied!"
 	},
+	"COMBOBOX": {
+		"PLACEHOLDER": "Filter...",
+		"CLEAR_SELECTIONS": "Clear all selected items",
+		"CLEAR_SELECTED": "Clear selected item",
+		"A11N": {
```
Comment (cal-smith) [inline src/i18n/en.ts]: Should be

```suggestion
		"A11Y": {
```

Or "ACCESSIBILITY" written out in full

Comment (rapzo) [reply src/i18n/en.ts]: LOL. What was i thinking? L11N and A11Y messed up my brain 😆 
Fixed.

**Pull request merged**